packages/core/src/js/feedback/FeedbackForm.tsx

```diff
@@ -90,6 +99,17 @@ export class FeedbackForm extends React.Component<FeedbackFormProps, FeedbackFor
     }
   };
 
+  public addRemoveScreenshot: () => void = () => {
```

Comment:
l: I find the name a bit confusing. Does `toggleScreenshot` or `onScreenshotButtonPress` should better to you?

Reply:
Good catch 👍 
Updated with 91a4b8ad95e4d539a134bb232f8d1fccc54bbfe1 to use `onScreenshotButtonPress`